src/Components/Server/src/Microsoft.AspNetCore.Components.Server.csproj

```diff
@@ -51,6 +51,10 @@
     <Compile Include="$(MessagePackRoot)SequenceReaderExtensions.cs" LinkBase="BlazorPack\MessagePack" />
   </ItemGroup>
 
+  <ItemGroup Label="Sources packages">
```

Comment:
? Does it really matter?

Reply:
@aspnet/build does it?

Reply:
I don't see any reason not to put this up w/ the other `<Reference>`s. Labels don't matter much here because automation won't be changing the file.